ledger/src/blockstore_db.rs

```diff
@@ -1323,11 +1334,11 @@ impl Column for columns::ErasureMeta {
         )
     }
 
-    fn index(key: &[u8]) -> (Slot, u64) {
-        let slot = BigEndian::read_u64(&key[..8]);
-        let set_index = BigEndian::read_u64(&key[8..]);
-
-        (slot, set_index)
+    fn index(key: &[u8]) -> Self::Index {
+        convert_key_bytes!(key,
+            0..8  => Slot::from_be_bytes,
+            8..16 => u64::from_be_bytes,  // FEC set index
```

Comment:
nit: Do you think the `// FEC set index` comment might be better served next to the type definition ?
```rust
type Index = (Slot, u64);
```
Similar comment for `MerkleRootMeta` & `ShredData` too

Reply:
added the comment to type definition as well.